src/forms/FormField.js

```diff
@@ -220,50 +218,66 @@ export default class FormField extends Component {
           <input
             className="field-description"
             onChange={onDescriptionChange}
-            defaultValue={ field.description }
-            style={ styles.fieldDescription }
+            defaultValue={field.description}
+            style={[styles.inputField, styles.fieldDescription]}
             type="text"
             placeholder="Description text (optional)" />
         </div>
 
         { this.getFieldEditor() }
 
         <div style={ styles.bottomButtons }>
-          <button className="field-close-button" style={ styles.cancelButton } onClick={onCancelClick}><FaClose /> Cancel</button>
-          <button
-            className="field-close-button save-button"
-            style={ [ styles.saveButton, field.error ? styles.saveButton.disabled : null ] }
+          <CoralButton
+            className="field-close-button"
+            icon="clear"
+            type="white"
+            onClick={onCancelClick}
+            style={{ marginRight: 10 }}
+          >
+            Cancel
+          </CoralButton>
+
+          <CoralButton
+            className="save-button"
+            type="success"
+            icon="done"
             onClick={onSaveClick}
-            disabled={ field.error ? 'disabled' : '' }
+            disabled={field.error}
           >
-            <FaFloppyO />
             Save
-          </button>
+          </CoralButton>
         </div>
-
       </div>
     );
   }
-
 }
 
 export const styles = {
+  base: {
+    fontFamily: 'Roboto'
```

Comment:
Or, even easier -- create a field in `settings` so if we ever change our minds (or the publisher does) they don't have to perform a text search to update all the fonts.